packages/react/src/components/SuiteHeader/util/IdleTimer.js

```diff
@@ -104,7 +104,8 @@ class IdleTimer {
     window.addEventListener('mousemove', this.eventHandler);
     window.addEventListener('mousedown', this.eventHandler);
     window.addEventListener('scroll', this.eventHandler);
-    window.addEventListener('keydown', this.eventHandler);
+    window.addEventListener('keydown', this.eventHandler,true);
```

Comment:
By changing this useCapture option we seem to be doubling the amount of fires. Our unit test are failing.